engine/class_modules/sc_death_knight.cpp

```diff
@@ -9064,7 +9151,7 @@ void death_knight_t::default_apl_frost()
   aoe -> add_talent( this, "Glacial Advance", "if=talent.frostscythe.enabled" );
   aoe -> add_action( this, "Frost Strike", "target_if=max:(debuff.razorice.stack+1)%(debuff.razorice.remains+1)*death_knight.runeforge.razorice,if=cooldown.remorseless_winter.remains<=2*gcd&talent.gathering_storm.enabled" );
   aoe -> add_action( this, "Howling Blast", "if=buff.rime.up" );
-  aoe -> add_action( this, "Death and Decay", "if=covenant.night_fae&spell_targets.death_and_decay>=2");
+  aoe -> add_action( "any_dnd,if=covenant.night_fae&spell_targets.death_and_decay>=2");
```

Comment:
In my opinion, calling death's due specifically and removing the night fae covenant condition would be just as good here.